addons/jam/magwells_762x39.hpp

```diff
@@ -19,3 +19,5 @@
 
     class CBA_762x39_SKS {};            // 762x39mm SKS magazines
     class CBA_762x39_SKS_Clip {};       // 762x39mm SKS clip
+
+    class CBA_762x39_CZ {};             // 762x39mm CZ 807 magazines
```

Comment:
This should be sorted alphabetically and given a better classname such as _CZ_807 to deconflict it from other CZ guns in 762x39 such as the CZ 527.